Therapeut Vechter/Assets/Scripts/UI/PlayTutorial.cs

```diff
@@ -0,0 +1,62 @@
+using FMOD.Studio;
+using FMODUnity;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PlayTutorial : MonoBehaviour
+{
+    private EventInstance dialogueAudioEventInstance;
+    [SerializeField]
+    private Texture[] image;
+    public FMODUnity.EventReference[] SoundToPlay;
+    public Button NextButton;
+    public RawImage tutorialRotater;
+
+    private int i;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    void Update()
```

Comment:
definitely unnecessary. this will make a script update every frame that does not need to update